recipes/flakeheaven/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "flakeheaven" %}
+{% set version = "0.11.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flakeheaven-{{ version }}.tar.gz
+  sha256: 20f5a573b8e85be5a73fed2e3967f7ab8b5e77714a5898d8b634dd31a0b75f9b
+
+build:
+  entry_points:
+    - flake8heavened = flakeheaven:flake8_entrypoint
+    - flakeheaven = flakeheaven:entrypoint
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry
```

```suggestion
    - flit-core >=2,<4
```
Are you sure that poetry is needed? This is specified upstream.

https://github.com/flakeheaven/flakeheaven/blob/3f0afc179dbd0907230fb3123f2938b0b63d3b7f/pyproject.toml#L179

---

Good catch. I was using `grayskull` to generate this `meta.yaml` file automatically so maybe it got a bit confused.

---

Yes, pyproject.toml support is not yet implemented: https://github.com/conda-incubator/grayskull/issues/322

---

Ah I see, good to know for next time!

---

Ok seems like upstream didn't specify their requirements correctly. Can you please add poetry back and file an issue upstream so that it can be fixed?

---

Hmm, looking at it, I think they've properly switched from using `flit` to `poetry` already at https://github.com/flakeheaven/flakeheaven/pull/24, just that there's no `0.11.1` tag on their GitHub repo (c.f. https://github.com/flakeheaven/flakeheaven/issues/63)

---

Ahh that's why, good catch. Then the python version needs to be updated again. Sorry, was looking at the wrong tag.